website/docs/docs/trusted-adapters.md

```diff
@@ -39,3 +31,12 @@ The following are **Trusted adapters** ✓ you can connect to in dbt Core:
 import AdaptersTrusted from '/snippets/_adapters-trusted.md';
 
 <AdaptersTrusted />
+
```

Comment:
The spacing here in the preview is a little wonky. It would be easy to miss the break
<img width="946" alt="Screenshot 2024-02-12 at 12 01 55 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/ca041b2b-ae23-4bef-8037-f5a3f19483b9">